apps/playlist/kinds/manifest.cue

```diff
@@ -3,7 +3,15 @@ package playlist
 manifest: {
 	appName:       "playlist"
 	groupOverride: "playlist.grafana.app"
-	kinds: [
-		playlist,
-	]
+	versions: {
+		"v0alpha1": {
+			codegen: {
+				frontend: false
```

Comment:
@IfSentient this seems to be ignored when using the codegen from the custom routes branch